Source/rclUE/Public/ROS2ActionClient.h

```diff
@@ -44,19 +44,10 @@ class RCLUE_API UROS2ActionClient : public UROS2Action
     rmw_request_id_t result_res_id;
     rmw_request_id_t cancel_res_id;
 
-    UPROPERTY()

```

Comment:
Don't need UPROPERTY()?

Reply:
Yeap, regular delegate, due to not being exposable to BP, does not support it :(